### ISSUE TITLE
Reindex tabs more often

### ISSUE THREAD

**Issue by Nattyfrank** [viktorgullmark/exilence-next#570]
I had my Currency tab in position 3 and a premium stash tab in position 4 (indexing starts at 0) when I added my account to Exilence Next and did the snapshot.  I switched positions of the tabs (aesthetics), so Currency was p4 and premium was p3, and Exilence next is saying that all my currency is in the premium tab and I have nothing in the currency tab.  

I removed and added the tabs again in the settings, closed and opened POE client.  Restarting the Exilence Next client did fix the issue though it appears.

**Comment by korey-enright** [comment]
I also experienced this issue, and restarting Exilence Next did fix the issue. I did not check if signing out from Exilence Next (in the top right corner) fixed the issue, but I imagine it most likely would. It appears that either GGG or Exilence Next is using the ordinal number of the tabs to send or receive data about what is in the tabs, and Exilence Next caches this and doesn't query the location of the tabs upon taking a snapshot.
